packages/shared/src/typeUtils.ts

```diff
@@ -5,7 +5,7 @@ export type UnionToIntersection<U> = (
   : never
 
 // make keys required but keep undefined values
-export type LooseRequired<T> = { [P in string & keyof T]: T[P] }
+export type LooseRequired<T> = { [P in keyof T]: T[P] }
```

Comment:
This makes `LooseRequired<T>` equivalent to `T`. The original `LooseRequired<T>` is different because all keys of `T` are not optional. It is added to match https://github.com/vuejs/rfcs/pull/293.

Reply:
See the difference [here](https://www.typescriptlang.org/play?#code/C4TwDgpgBAMg9nAzhAShAjgVwJYCcIAmAPACoB8UAvFAN5QDaAClNgHZSLC5sDmUAZFADWEEHABmUEgF0AXFKbSoAXwBQq0JCiNccMIiq1VUE1HEJ5nbqx4AaY6YBGAQ1wB+eY4QAbCM9aqaqrimKwAxsDYcOzAEJwAjFAAFGC6+vI6eogAlLRqIeGR0VCxnABMyalZ8vBIqBg4+MSZ+mS5NEGlwPFJdOZw8gDkgyrZGnHAZb1mFlDDo+Pl0-1Dg7ZQLrjyoQQQ4myEo0A).